main_test.go

```diff
@@ -19,6 +13,11 @@ import (
 	"github.com/Financial-Times/transactionid-utils-go"
 	"github.com/gorilla/mux"
 	"github.com/stretchr/testify/assert"
+	"io/ioutil"
```

Comment:
You can use `goimports` linter to make sure that the imports are in the following order:
- imports from the Go standard library in alphabetical order;
- external imports in alphabetical order;
Every go code file follows these rules.